README.md

```diff
@@ -29,6 +29,14 @@ We use these to make it compatible with running the same script from Lambda and
 	AWS_LAMBDA_FUNCTION_NAME - set if running in lambda(Automatically set in Lambda)
 	GET_FORECAST_AWS_PROFILE - set for testing on command line to pick a profile from your credentials file
 
+
+### Github Secrets for S3 Upload
```

Comment:
we need another readme for github_actions_Readme.md that has instructions how to configure github, we should do it security not with credentials, not sure what is possible